src/main/java/mat/server/CQLLibraryService.java

```diff
@@ -1000,9 +1000,11 @@ public SaveUpdateCQLResult saveCQLFile(String libraryId, String cql) {
             CQLLibrary cqlLibrary = cqlLibraryDAO.find(libraryId);
             String cqlXml = getCQLLibraryXml(cqlLibrary);
 
+            boolean isFhir = cqlLibrary.isFhirLibrary();
             CQLLinterConfig config = new CQLLinterConfig(cqlLibrary.getName(),
-                    MeasureUtility.formatVersionText(cqlLibrary.getRevisionNumber(), cqlLibrary.getVersion()),
-                    cqlLibrary.getLibraryModelType(), cqlLibrary.isFhirMeasure() ? cqlLibrary.getFhirVersion() : cqlLibrary.getQdmVersion());
+                    MeasureUtility.formatVersionText(cqlLibrary.getRevisionNumber(),cqlLibrary.getVersion()),
+                    isFhir ? "FHIR" : QDMUtil.QDM_MODEL_IDENTIFIER ,
+                    isFhir ? cqlLibrary.getQdmVersion() : cqlLibrary.getFhirVersion());
```

Comment:
Should `isFhir ? cqlLibrary.getQdmVersion() : cqlLibrary.getFhirVersion()` be
`isFhir ? cqlLibrary.getFhirVersion(): cqlLibrary.getQdmVersion()`

Reply:
good catch. Fixed